data/outersys.ssc

```diff
@@ -530,7 +546,7 @@ ReferencePoint "Lempo-Hiisi" "Sol"
 # Colors from Grundy et al. (2015), Icarus 257, p. 130-138
 # "The mutual orbit, mass, and density of the large transneptunian binary system Varda and Ilmarë"
 # https://ui.adsabs.harvard.edu/abs/2015Icar..257..130G/abstract
-ReferencePoint "Varda-Ilmare" "Sol"
+ReferencePoint "Varda-Ilmar\u00EB" "Sol"

```

Comment:
Is there a need to do this on current versions of Celestia? AFAIK, with the intrduction of TTF fonts, it should be possible to input special characters directly in data files.

Reply:
No idea. I can definitely just go with inputting the special character, but I don't know if that breaks stuff somewhere.